### PR TITLE
Add release notes for v7.11.1

### PR DESCRIPTION
**Pull request by pugnascotia**
Add release notes for 7.11.1. I omitted PRs for Autoscaling and the fix to the LLRC license headers.

I didn't explicitly call out the LDAP fix, but I could move it to the top if we prefer?

### CODE REVIEW

File: docs/reference/release-notes/7.11.asciidoc [hunk 1]
```diff
@@ -1,8 +1,82 @@
...
+=== Enhancements
+
+Features/Indices APIs::
+* Prevent concurrent rollover race conditions {es-pull}67953[#67953] (issues: {es-issue}64921[#64921], {es-issue}67836[#67836])
+
+Features/Stats::
+* Add index creation version stats to cluster stats {es-pull}68141[#68141]
+
+Snapshot/Restore::
+* Improve shards evictions in searchable snapshot cache service {es-pull}67160[#67160] (issues: {es-issue}66730[#66730], {es-issue}66958[#66958])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.11.asciidoc:17]:
```suggestion
* Improve shard evictions in searchable snapshot cache service {es-pull}67160[#67160] (issues: {es-issue}66730[#66730], {es-issue}66958[#66958])
```

File: docs/reference/release-notes/7.11.asciidoc [hunk 2]
```diff
@@ -1,8 +1,82 @@
...
+* Fix searchable snapshot action to wait for the restored index {es-pull}68309[#68309] (issue: {es-issue}67879[#67879])
+* Ignore `clusterChanged` events if state not recovered {es-pull}67507[#67507] (issue: {es-issue}67506[#67506])
+* `ShrinkStep` ignores the `isAcknowledged` flag on the response {es-pull}67591[#67591] (issue: {es-issue}53111[#53111])
+* Skip rollover if the data stream is rolled over already {es-pull}67778[#67778] (issue: {es-issue}67777[#67777])
+
+Features/Indices APIs::
+* Fix exception when validating composable template with no template {es-pull}67310[#67310] (issue: {es-issue}66949[#66949])
+
+Features/Ingest::
+* Fix parsing empty value with brackets failed in Dissect ingest processor {es-pull}65524[#65524] (issue: {es-issue}65080[#65080])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.11.asciidoc:44]:
```suggestion
* Fix parsing empty value with brackets failed in dissect ingest processor {es-pull}65524[#65524] (issue: {es-issue}65080[#65080])
```

File: docs/reference/release-notes/7.11.asciidoc [hunk 3]
```diff
@@ -1,8 +1,82 @@
-[[release-notes-7.11.0]]
-== {es} version 7.11.0
+[[release-notes-7.11.1]]
+== {es} version 7.11.1
 
 Also see <<breaking-changes-7.11,Breaking changes in 7.11>>.
 
+[[enhancement-7.11.1]]
```
Comment (jrodewig) [inline docs/reference/release-notes/7.11.asciidoc:6]: I don't think we need to mention the LDAP issue as it's already covered in the 7.11.0 notes:
https://www.elastic.co/guide/en/elasticsearch/reference/7.x/release-notes-7.11.0.html#known-issues-7.11.0

**Review by jrodewig — APPROVED** [review]
LGTM. I pushed 6db7356335beb6523d357f68f5c276ae1dfb64e9 to add an xref to the main release notes page.

I also added the `v7.12.0` label. We'll want to backport these changes to the 7.x branch.

File: docs/reference/release-notes/7.11.asciidoc [hunk 4]
```diff
@@ -1,8 +1,82 @@
-[[release-notes-7.11.0]]
-== {es} version 7.11.0
+[[release-notes-7.11.1]]
+== {es} version 7.11.1
```
Comment (jrodewig) [inline docs/reference/release-notes/7.11.asciidoc:2]: We'll need to add a coming tag if we merge this before the release day.
```suggestion
== {es} version 7.11.1

coming::[7.11.1]

```

File: docs/reference/release-notes/7.11.asciidoc [hunk 5]
```diff
@@ -1,3 +1,85 @@
...
+* Remove leniency for casting from a `def` type to a `void` type in Painless {es-pull}66957[#66957] (issue: {es-issue}66175[#66175])
+
+Machine Learning::
+* Ensure mappings are up to date before reverting state {es-pull}68746[#68746]
+
+Network::
+* Fix `AbstractClient#execute` Listener Leak {es-pull}65415[#65415] (issue: {es-issue}65405[#65405])
+
+Packaging::
+* Fix installation using Windows MSI (issue: {es-issue}68914[#68914])
```
Comment (pugnascotia) [inline docs/reference/release-notes/7.11.asciidoc:69]: @jrodewig I added this for https://github.com/elastic/elasticsearch/issues/68914 / https://github.com/elastic/windows-installers/issues/402

Comment (jrodewig) [reply docs/reference/release-notes/7.11.asciidoc:69]: Thanks @pugnascotia! I'll reach out offline.

Comment (pugnascotia) [reply docs/reference/release-notes/7.11.asciidoc]: @jrodewig a fix is merged - I've linked to the PR in the relevant repo.

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks @pugnascotia.

**Pull request merged**